src/jit/codegenarm64.cpp

```diff
@@ -5778,7 +5777,8 @@ void CodeGen::genPutArgStk(GenTreePutArgStk* treeNode)
                                                // as that is how much stack is allocated for this LclVar
                 isHfa          = varDsc->lvIsHfa();
                 gcPtrCount     = varDsc->lvStructGcCount;
-                structGcLayout = varDsc->lvGcLayout;
+                for(unsigned i = 0; i < gcPtrCount; ++i)
+                  gcPtrs[i] = varDsc->lvGcLayout[i];
```

Comment:
It is under found at tests\scripts\format.py in the coreclr repo. An example of its use is:

<pre>
python -u tests\scripts\format.py -c /home/jashoo/coreclr -o Windows_NT -a x64
</pre>